ext/gmp/tests/gh16870.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+GH-16870 (gmp_pow(64, 11) throws overflow exception)
+--EXTENSIONS--
+gmp
+--FILE--
+<?php
+var_dump((string) gmp_pow(64, 11));
+var_dump((string) gmp_pow(-1, PHP_INT_MAX));
+var_dump((string) gmp_pow(0, PHP_INT_MAX));
+var_dump((string) gmp_pow(1, PHP_INT_MAX));
+var_dump((string) gmp_pow("-1", PHP_INT_MAX));
+var_dump((string) gmp_pow("0", PHP_INT_MAX));
+var_dump((string) gmp_pow("1", PHP_INT_MAX));
+var_dump((string) gmp_pow(3, 10094));
```

Comment:
Please add tests that use the `**` operator, as those would also be affected I guess? Or did we somehow not covered this in the initial bug fix?